lib/spack/docs/getting_started.rst

```diff
@@ -600,6 +600,61 @@ flags to the ``icc`` command:
            spec: intel@15.0.24.4.9.3
 
 
+^^^^^^^^^^
+SYCL/DPC++
+^^^^^^^^^^
```

Comment:
I don't think this belongs in the "getting started" section.

Maybe it could go in the longer section on intel tools?

Reply:
https://spack.readthedocs.io/en/latest/build_systems/intelpackage.html?highlight=intel#intel-packages-in-spack

Reply:
Wasn't sure, that section reads:
```
Vendor-Specific Compiler Configuration

With Spack, things usually “just work” with GCC. Not so for other compilers. This section provides details on how to get specific compilers working.
```

Reply:
I struggled with DPCPP just last week so this is well timed.

Reply:
@frankwillmore Can you look at the compilers.yaml?

Reply:
@rscohn2 That stanza looks pretty involved for an example. I've never had to add that much stuff, by ymmv. 
I stopped naming the compiler as 'clang' a while back, in favor of 'oneapi', because I think it's important to distinguish the intel version, as with the cray version of it. All of the environment stuff may well be necessary and valid for that particular installation, and I don't see anything fundamentally wrong, but imho it looks a bit cluttered. 

In my work, I've used icpx instead of dpcpp. afaik it will compile dpcpp code, just needs a few extra include paths. And I would set up the environment in a module, but this may just be a matter of taste.

Reply:
I also stopped using dpcpp driver and instead add -fsycl to compile and link. All the cmake docs discourage changing the driver.
Unfortunately, you may also need -fscyl-unnamed-lambda for compile, but I hope they make that unnecessary.